python/beeai_framework/tools/mcp_tools.py

```diff
@@ -52,16 +43,17 @@ def is_empty(self) -> bool:
         return not self.result
```

Comment:
This implementation is basically `JSONToolOutput` in TypeScript. Can you add it and reuse it here?